hibike/setup.sh

```diff
@@ -37,5 +37,13 @@ if [ "$(dpkg-query -W -f='${Status}' avr-libc 2>/dev/null | grep -c "ok installe
 	apt-get --assume-yes install avr-libc	
 fi
 
+if [ "$(dpkg-query -W -f='${Status}' python-pip 2>/dev/null | grep -c "ok installed")" != 1 ]; then
+	apt-get --assume-yes install python3-pip
+fi
+
+if [ "$(pip3 freeze | grep -c "pyserial")" != 1]; then
+        pip3 install -r requirements.txt
```

Comment:
I have no idea how you even catch these errors. GIVE ME YOUR VIMRC

Good catch